tuf/api/metadata.py

```diff
@@ -454,22 +489,31 @@ def to_dict(self) -> JsonDict:
         """Returns the JSON-serializable dictionary representation of self. """
         json_dict = super().to_dict()
         json_dict.update({
-            'meta': self.meta
+            'meta': {
+                'snapshot.json': self.meta['snapshot.json'].to_dict()
```

Comment:
Nit: we might want to introduce constants for things like the role names and the file extensions, 'meta', etc.

Reply:
I agree we need a module containing different constants. 
I will add an issue for that so we can discuss it as a change in a future pr.